### PR TITLE
Update funding-your-training.md

### PR DESCRIPTION
**Pull request by rosecatt**
Proposal for Funding page content based on finance guidance page being deleted

### Trello card
https://trello.com/c/AlGm6fI3/1387-content-audit

### Context
We've buried some finance content quite far down, and it can be difficult to surface for users.

### Changes proposed in this pull request
Moving existing, approved, content up one level to better serve user needs and remove duplication.

### CODE REVIEW

**Comment by rosecatt** [comment]
Hello, I wasn't sure how to attach the right CTA design modules to the right part of the page - so this page is not looking at it should do right now.

**Comment by peteryates** [comment]
I can help with the right column content @rosecatt

File: content/funding-your-training.md [hunk 1]
```diff
@@ -101,29 +112,33 @@ below.
...
+### Non-graduate bursaries
 
-Even if a bursary or scholarship is available for your course, you can get student finance to cover your fees and help with your living costs. You can apply for a tuition fee and/or a maintenance loan even if you already have a student loan.
+If you're a final year undergraduate student and your course started in the academic year 2021 to 2022, you may be eligible for a training bursary of £9,000. The bursary is available if you're studying a QTS course in secondary maths or physics, or you're studying an opt-in QTS course in secondary undergraduate computing, languages, mathematics or physics.
 
-You can apply for a tuition fee loan of up to £9,250 to cover the full cost of your course fees and a maintenance loan of up to £12,382 to help with your living costs.
+There are [Troops to Teachers] (https://www.ucas.com/teaching-option/troops-teachers) tax-free bursaries of £40,000 for former military personel  undertaking  undergraduate degrees leading to Qualified Teacher Status (QTS) in England. You may be eligible for the bursary if you train to teach secondary biology, physics, chemistry, computing, maths or modern foreign languages.
```
Comment (peteryates) [inline content/funding-your-training.md]: There's a space here between the link text and target, it should be `[Troops to Teachers](https://something)`

```suggestion
There are [Troops to Teachers](https://www.ucas.com/teaching-option/troops-teachers) tax-free bursaries of £40,000 for former military personel  undertaking  undergraduate degrees leading to Qualified Teacher Status (QTS) in England. You may be eligible for the bursary if you train to teach secondary biology, physics, chemistry, computing, maths or modern foreign languages.
```

File: content/funding-your-training.md [hunk 2]
```diff
@@ -74,20 +74,31 @@
     - Isle of Man
 ---
 
+
+## Tuition fee and maintenance loans
+
+You can apply for a tuition fee loan of up to £9,250 to cover your teacher training, so you don't need to pay course fees upfront. You can also apply for a maintenance loan of up to £12,382 to help with living costs.
+
+You can still apply for a tuition fee and/or a maintenance loan if you already have a student loan, and regardless of whether you get a bursary or scholarship.
```
Comment (peteryates) [inline content/funding-your-training.md]: Would 'or' make sense here instead of 'and/or'?

**Review by peteryates — COMMENTED** [review]
This looks like a great start, one minor error to fix and I left a question. Nice work!

**Comment by MylesJarvis** [comment]
Thanks @peteryates. @rosecatt, in terms of the detailed finance guidance page, am I right in thinking we'll be bringing some of that over in this PR, or will that be modified on a separate PR?

File: content/funding-your-training.md (hunk 2 above)
Comment (MylesJarvis) [reply content/funding-your-training.md]: I actually think 'and' is more appropriate. I believe most people will get both of these as opposed to just one.

**Comment by MylesJarvis** [comment]
@rosecatt @peteryates - I'm going both approve and ship this on Monday, we need to ensure our VWO traffic redirects are pointing here instead of our detailed funding page (even though we're not deleting the page in this PR).

**Review by MylesJarvis — APPROVED** [review]
LGTM

**Pull request merged**